taskt/Core/Automation/Commands/Data Commands/GetPDFTextCommand.cs

```diff
@@ -1,81 +1,81 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Net;
 using System.Windows.Forms;
 using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
 using taskt.Core.IO;
+using taskt.Core.Utilities.CommonUtilities;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
-using taskt.Core.Utilities.CommonUtilities;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Data Commands")]
-    [Attributes.ClassAttributes.Description("This command allows user to get text from a PDF File")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to get all text from a PDF File into a variable")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Data Commands")]
+    [Description("This command reads all text from a PDF file and saves in a variable.")]
     public class GetPDFTextCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please indicate the PDF file path or PDF file URL")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowFileSelectionHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter or Select the path to the applicable file or enter file URL.")]
-        [Attributes.PropertyAttributes.SampleUsage(@"C:\temp\myfile.pdf , [vFilePath] or https://temp.com/myfile.pdf")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_FilePath { get; set; }
+        [PropertyDescription("Source Type")]
+        [PropertyUISelectionOption("File Path")]
+        [PropertyUISelectionOption("File URL")]
+        [InputSpecification("Select source type of PDF file.")]
+        [SampleUsage("")]
+        [Remarks("Select 'File Path' if file is locally placed or 'File URL' to read file from a web URL.")]
```

Comment:
Change to "Select 'File Path' if the file is locally place or 'File URL' to read a file from a web URL."

Reply:
Changed to "Select 'File Path' if the file is locally **placed** or 'File URL' to read a file from a web URL."